### PR TITLE
Feat/adding questions to consultations

### PR DESCRIPTION
**Pull request by BALAJI-BALA**
Update Consultation reading time based on page builder component update

### CODE REVIEW

File: app/javascript/application/coffee_scripts/consultations.coffee [hunk 1]
```diff
@@ -77,4 +77,86 @@ $(document).on 'turbolinks:load', ->
...
+			$('#add-option-btn').removeClass("d-none")
+			$('.checkbox-option-row').removeClass("d-none")
+			$('.checkbox-option').addClass("d-none")
+			$('.radio-button-option').removeClass("d-none")
+		else
+			$('#add-option-btn').addClass("d-none")
+			$('.checkbox-option-row').addClass("d-none")
+	$(document).on 'change', '.edit_question .question_question_type select', ()->
+		if $(this).val() == "checkbox"
+			$(this).parent().parent().find('.edit-add-option').removeClass('d-none')
```
Comment (mikevic) [inline app/javascript/application/coffee_scripts/consultations.coffee:146]: @BALAJI-BALA - have you considered using `closest` instead of parent.parent?

Comment (BALAJI-BALA) [reply app/javascript/application/coffee_scripts/consultations.coffee:146]: @mikevic Bro the closest will find and return only one div even though if we want many div which has the same class name so in that case I am using find

Comment (BALAJI-BALA) [reply app/javascript/application/coffee_scripts/consultations.coffee:146]: @mikevic sorry bro the closest will find many div with same class_name, but the DIV structure is not in the way to use closest option

Comment (mikevic) [reply app/javascript/application/coffee_scripts/consultations.coffee:146]: okay, cool

**Pull request merged**